### PR TITLE
Support to add the SSL certificate in the simplivity trust store

### PR DESCRIPTION
**Pull request by svt-hchaudhari**
Signed-off-by: Hemant Chaudhari <hchaudhari@hpe.com>

### Description
Support to add the SSL certificate in the simplivity trust store

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: simplivity/resources/certificates.py [hunk 1]
```diff
@@ -51,6 +51,20 @@ def get_by_name(self, name):
...
+        Args:
+            certificate: Base64 encoded SSL certificate.
+            timeout: Time out for the request in seconds.
+
+        Returns:
+            dict : Returns the certificate details.
+        """
+
+        data = {"certificate": certificate}
+        return self._client.do_post(URL, data, timeout)
```
Comment (sijeesh) [inline simplivity/resources/certificates.py:66]: I think, we should return the created certificate object.

**Review by sijeesh — CHANGES_REQUESTED** [review]

File: simplivity/resources/certificates.py (hunk 1 above)
Comment (svt-hchaudhari) [reply simplivity/resources/certificates.py:66]: Hi Sijeesh,
POST command returns the certificate object details

Comment (sijeesh) [reply simplivity/resources/certificates.py:66]: Got it. I was thinking if it returns a single certificate object details we can create an object of the certificate and return it to match with the other resource implementation.

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**